__tests__/integration.js

```diff
@@ -392,12 +392,31 @@ test('yarn run <script> <strings that need escaping>', async () => {
 
   const options = {cwd, env: {YARN_SILENT: 1}};
 
-  const trickyStrings = ['$PWD', '%CD%', '^', '!', '\\', '>', '<', '|', '&', "'", '"', '`', '  '];
+  const trickyStrings = ['$PWD', '%CD%', '^', '!', '\\', '>', '<', '|', '&', "'", '"', '`', '  ', '()'];
```

Comment:
Why does this string need to be added?

Reply:
Because parenthesis are also special characters and we were not testing them. We don't have to add it for this specific PR tho.

Reply:
Shouldn't they be in separate string? `'(', ')'`?

Reply:
Sure :)

Reply:
Done!